kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -884,6 +884,13 @@ object KyuubiConf {
       .timeConf
       .createWithDefault(Duration.ofMinutes(30).toMillis)
 
+  val SERVER_STATE_STORE_RECOVERY_PER_BATCH: ConfigEntry[Int] =
+    buildConf("kyuubi.server.state.store.recovery.per.batch")
+      .doc("The number to recover from state store per batch.")
```

Comment:
The number <of what>? I'm sorry I can not get your point from the conf name and desc.